### PR TITLE
Aggregation: be able to return a complete row sample as default aggregation

### PR DESCRIPTION
**Pull request by dgaubert**
- Support default `full-sample` aggregation for PostGIS vector renderer
- Remove exited containers after running test against docker
- Link to windshaft#pg-mvt-do-not-filter-columns

Do NOT merge yet: wait for https://github.com/CartoDB/Windshaft/pull/583

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "step-profiler": "~0.3.0",
     "turbo-carto": "0.20.2",
     "underscore": "~1.6.0",
-    "windshaft": "4.1.1",
+    "windshaft": "cartodb/windshaft#pg-mvt-do-not-filter-columns",
```
Comment (rochoa) [inline package.json]: I guess we can start using 4.2.0 as I just published it.

Comment (dgaubert) [reply package.json]: Nice!

**Comment by dgaubert** [comment]
Windshaft 4.2.0 released and updated in this PR. Shall we merge it?

**Comment by rochoa** [comment]
:shipit:

**Pull request merged**